src/main/java/org/heigit/ohsome/ohsomeapi/config/SwaggerConfig.java

```diff
@@ -73,6 +73,18 @@ public Docket rawDataDocket() {
                 "Direct access to the full-history of the OSM data"))
         .forCodeGeneration(true).globalResponseMessage(RequestMethod.GET, responseMessages);
   }
+  
+  /** Creates the Swagger2 documentation for the contributions resources. */
```

Comment:
ah true.. yeah it's probably alphabetically.. I will check if there's a way to modify that, otherwise I could add a number in front, like "1 - Data Aggregation" "2 - Data Extraction" etc.

Reply:
Can't find any function to specify a custom ordering. So then I'll just go with the proposed schema with adding a number in front, followed by a minus (-) so it's a bit more clear that this is used for ordering:

1 - Data Aggregation
2 - Data Extraction
3 - Contributions
4 - Metadata

Reply:
not sure if I like the solution with number prefixes too much, since it would break existing deep links to specific specs (e.g. https://api.ohsome.org/v1/swagger-ui.html?urls.primaryName=Metadata) :thinking: 

I found this "solution" on SO: https://stackoverflow.com/a/60329961/1627467 – would something like that work for us?

Reply:
Puh.. that's really a "solution" yeah :D don't really like that one either, as it's a bit hacky as also described in the comment.
If you are really against the numbering, we could then also keep it now as-is and add the specific links to the different speccs then wherever it's needed, instead of the generic one. And I think the idea is still to replace swagger at some point with a more sophisticated playground, so I would not put too much time in swagger anyway.

Reply:
> against the numbering

yeah, breaking existing links is not very good practice, if you ask me.

> https://stackoverflow.com/a/60329961/1627467

this wasn't too bad after all: 64247014dbbe40a2b8fb9c97744867ce601bc183
why would you consider it to be hacky? it just implements the interface (`SwaggerResourcesProvider`) which is AFAICT meant to be used for situations like this one.